yt_dlp/extractor/twitcasting.py

```diff
@@ -38,7 +38,7 @@ class TwitCastingIE(InfoExtractor):
             'description': 'Twitter Oficial da cantora brasileira Ivete Sangalo.',
             'thumbnail': r're:^https?://.*\.jpg$',
             'upload_date': '20110822',
-            'timestamp': 1314010824,
+            'timestamp': 1313978424,
```

Comment:
How come timestamp changed?

Reply:
Not sure. It looks like the webpage format had changed slightly, I also had to fix the `view_count` extraction

Reply:
![image](https://user-images.githubusercontent.com/10355528/209984417-72050c48-07a3-4dec-8285-ea78c2a1d3e7.png)
![image](https://user-images.githubusercontent.com/10355528/209984449-22ad05a7-f723-42f9-91ed-dd3f11de08d4.png)
Looks like they seem to store the timestamp by UTC now? (the difference is exactly 9 hours)